tex/texlive-bin/Portfile

```diff
@@ -253,6 +253,20 @@ if {[string match *clang* ${configure.cxx}]} {
     configure.cxxflags-append -Wno-reserved-user-defined-literal
 }
 
+destroot.violate_mtree  yes
+
+# Useful variables to set up the TeXDist directory structure.
+# See below in post-destroot and pre-activate.
+set td_root         /Library/TeX/Distributions
+set td_fd           /.FactoryDefaults
```

Comment:
I would remove the leading slash and put it in `td_mpfd` variable instead. The note about more verbose names still applies.